### ISSUE TITLE
notifications bug

### ISSUE THREAD

**Issue by lostamongtrees** [RolePlayGateway/roleplaygateway#177]
was brought here
![image](https://user-images.githubusercontent.com/52435234/65646778-378ebd00-dfca-11e9-8dda-143ecaf972a7.png)

**Comment by lostamongtrees** [comment]
ok and when i can access notifications, everywhere i click leads to 

![image](https://user-images.githubusercontent.com/52435234/65646798-4e351400-dfca-11e9-90f9-7e05ad6b91f7.png)


![image](https://user-images.githubusercontent.com/52435234/65646809-54c38b80-dfca-11e9-9d00-2557286ff448.png)
![image](https://user-images.githubusercontent.com/52435234/65646814-5a20d600-dfca-11e9-8f0d-d557329809c5.png)

![image](https://user-images.githubusercontent.com/52435234/65646833-660c9800-dfca-11e9-9abe-71fbedcd2293.png)

**Comment by lostamongtrees** [comment]
seems t be fixed cannot replicate / will reopen if occurs again
